Add on-demand metrics for alerts (#7711)
getsentry/sentry-docs: src/docs/product/alerts/create-alerts/metric-alert-config.mdx

### Tags & Properties

Add filters in the provided input to narrow down what you'll be alerted about, such as transaction tags or other event properties. Available properties depend on your alert's event type:

> - for **error** events, all error properties are available. See [Searchable Properties](/product/sentry-basics/search/searchable-properties/#event-properties) for a full list

Reviewer: ```suggestion
- For **error** events, all error properties are available. See [Searchable Properties](/product/sentry-basics/search/searchable-properties/events/#searchable-properties) for a full list.
```

Need to update the link since searchable properties was broken down into 4 child pages. Not sure if there's a better link for error event properties?

> - for **transaction** events, the following basic properties are generally available: `release`, `transaction`, `transaction.status`, `transaction.op`, `http.method`, `http.status_code`, `os.name`, `browser.name` and `geo.country_code`.

Reviewer: ```suggestion
- For **transaction** events, the following basic properties are generally available: `release`, `transaction`, `transaction.status`, `transaction.op`, `http.method`, `http.status_code`, `os.name`, `browser.name`, and `geo.country_code`.
```

#### Advanced filters for Transactions

<Note>

This feature is only available to organizations with a Business or Enterprise plan.

</Note>

> Filtering metric alerts by additional tags or properties is possible, however we will only start collecting these metrics once the alert is saved. If such a field is used, we will provide an estimation for the previous period during alerts creation.

Reviewer: ```suggestion
You can add filters by tags or properties to your metric alerts. However, Sentry only starts collecting these metrics after the alert is saved. If such a filter is used, Sentry provides an estimate of the metric for the previous period during the alert's creation.
```

Reworded to use "you" instead of "we" to be more consistent with the rest of the doc.

Author: ```suggestion
A wider range of fields or tags is available for filtering, beyond the basic transaction properties mentioned above. However, Sentry only starts collecting these metrics after the alert is saved. If such a filter is used, Sentry provides an estimate of the metric for the previous period during the alert's creation.
```

Author: @vivianyentran thanks for all the suggestions! I rephrased this one again - the idea is that customers on Team can filter alerts based on those few properties mentioned above. Customers on Business can actually access a much wider range of tags/fields. Hope this sounds better :)

> Filtering metric alerts by additional tags or properties is possible, however we will only start collecting these metrics once the alert is saved. If such a field is used, we will provide an estimation for the previous period during alerts creation.

Author: Yes, that should be a given at this point and we never introduced any concept of stored/total transactions here.
